resources/guides/uwinsite_guide.md

```diff
@@ -0,0 +1,236 @@
+---
+id: uwinsite_guide
+title: UWinsite Guide
+sidebar_label: UWinsite Guide
+slug: /guides/uwinsite_guide
+---
+
+This is a guide on how to use and access the important sections on your UWinsite Student page.
+
+## Course Management
+
+### How to Plan & Enroll in Classes
+
+To plan your courses and register in classes in UWinsite Student, you will use the
+**Manage Classes** section on the Student Homepage.
+
+#### Planning your Classes
+
+You can start planning your classes and adding them to your Shopping Cart in UWinsite
+Student when the validation period opens. The validation period is when the timetable
+for the next semester is close to final and the courses being offered are visible in
+UWinsite Student. Typically, the validation period starts two to three weeks before
+registration begins.
+
+The preferred method for placing your classes into the Shopping Cart is through Class
+Search and Enroll. This method shows you just the courses that are offered during the
+term for which you are planning.
+
+**To get started planning your classes:**
+
+![img](../../static/img/uwinsite_guide/uwinsite_homepage.jpg)
```

Comment:
the alt text(the part inside the `[..]`) should always be a title or description of the image. it is used by screen readers to help people who cannot see the image.

for example if I were to put a photo of a chair, I would write
```md
![photo of chair](../../static/img/chair.jpg)
```

BTW this goes for all images without alt text in this PR(and in general)